### PR TITLE
test(aio): more linting

### PR DESCRIPTION
**Pull request by gkalpak**
Start linting for `aio/transforms/` as part of `yarn lint` and add rules to guard against jasmine focused tests/suites for `aio/transforms/` and app unit and e2e tests.

### CODE REVIEW

**Comment by gkalpak** [comment]
@petebacondarwin, can you please make sure I didn't change/remove something that matters from transforms (e.g. some dependency imported/injected for its side effects)?

File: aio/transforms/target-package/services/targetEnvironments.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ module.exports = function targetEnvironments() {
 
   var updateActiveCount = function() {
     _activeCount = 0;
-    for (target in _targets) {
+    for (var target in _targets) {
```
Comment (petebacondarwin) [inline aio/transforms/target-package/services/targetEnvironments.js]: `let`?

**Review by petebacondarwin — APPROVED** [review]

File: aio/transforms/target-package/services/targetEnvironments.js (hunk 1 above)
Comment (gkalpak) [inline aio/transforms/target-package/services/targetEnvironments.js]: Even better :smiley: I was just trying to keep inline with the rest of the file.

Comment (gkalpak) [inline aio/transforms/target-package/services/targetEnvironments.js]: Done.

Comment (gkalpak) [inline aio/transforms/target-package/services/targetEnvironments.js]: (Here is the commit before it got squashed: 1820122)

**Pull request merged**